lib/randomfileaccess.js

```diff
@@ -134,10 +135,26 @@ class RandomFileAccess {
      */
     async write(path, range, buffer, totalSize) {
         const { fileHandles } = this[PRIVATE];
-        const handle = await getOrOpen(fileHandles, path, "w", totalSize);
+        const handle = await getOrOpen(fileHandles, path, FileFlags.WRITEONLY, totalSize);
```

Comment:
What do you think of the name `acquire` instead of `getOrOpen`?

Reply:
I would leave that up to @bobvanmanen.

Reply:
I think `getOrOpen` is more clear.